lib/ModuleFilenameHelpers.js

```diff
@@ -99,22 +99,31 @@ ModuleFilenameHelpers.createFilename = function createFilename(module, moduleFil
 		.replace(ModuleFilenameHelpers.REGEXP_HASH, hash);
 };
 
-ModuleFilenameHelpers.createFooter = function createFooter(module, requestShortener) {
+ModuleFilenameHelpers.createFooter = function createFooter(module, requestShortener, sourceUrlComment) {
```

Comment:
The `sourceUrlComment` is not part of the footer. It doesn't make sense to move it into this module.

Reply:
You're right, sorry. I've made the requested change.

What was the use case which required this feature? I don't see the point, but I also don't know much about Webpack's internals.